cli/integration_tests/edit_turbo_json/fixture-configs/1-baseline.json

```diff
@@ -0,0 +1,19 @@
+{
+  "$schema": "https://turbo.build/schema.json",
+  "globalDependencies": ["foo.txt"],
+  "globalEnv": ["SOME_ENV_VAR"],
+  "pipeline": {
+    "build": {
+      "env": ["NODE_ENV"],
+      "outputs": []
+    },
+    "my-app#build": {
+      "outputs": ["banana.txt", "apple.json"]
+    },
+
+    "something": {},
+    "//#something": {},
+
+    "maybefails": {}
```

Comment:
I don't think we need all these task definitions for the tests. Only `build` and `my-app#build` are maybe needed. Can we delete the extras?